### PR TITLE
Bump to k8s 1.18.3

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: pkg/scheduler/framework/framework.go [hunk 1]
```diff
@@ -139,7 +138,7 @@ func (f *framework) VolumeBinder() scheduling.SchedulerVolumeBinder {
 }
 
 func NewFrameworkHandle(pods []*v1.Pod, nodes []*v1.Node) v1alpha1.FrameworkHandle {
-	snapshot := k8scache.NewSnapshot(pods, nodes)
+	snapshot := cache.NewSnapshot(pods, nodes)
```
Comment (k82cn) [inline pkg/scheduler/framework/framework.go:142]: let's open an issue in upstream to make cache public.

Comment (hzxuzhonghu) [reply pkg/scheduler/framework/framework.go:142]: https://github.com/kubernetes/kubernetes/issues/91983

**Comment by hzxuzhonghu** [comment]
@k82cn please take a look at the scheduling part, especially the nodeorders.go and predicate.go

File: pkg/scheduler/framework/framework.go (hunk 1 above)
Comment (k82cn) [reply pkg/scheduler/framework/framework.go:142]: ok, let's move this PR forward; I don-t think we will have a conclusion in a shot time :(

Comment (k82cn) [reply pkg/scheduler/framework/framework.go:142]: ok, let's move this PR forward; I don-t think we will have a conclusion in a shot time :(

**Comment by k82cn** [comment]
> @k82cn please take a look at the scheduling part, especially the nodeorders.go and predicate.go

How can I find them in 1.4k files and 15 commits? please separate this PR to some meaningful commits, e.g. scheduler, controller, vendor, test ...

**Pull request merged**